.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's not commit IDE specific configurations. Could you add .vscode to .gitignore

Reply:
added in the latest revision